exercise1.Rmd

```diff
@@ -13,4 +13,41 @@ knitr::opts_chunk$set(
   collapse = TRUE,
   comment = "#>"
 )
+
+library(ggplot2)
+ba = read.table("https://raw.githubusercontent.com/OxfordIHTM/teaching_datasets/main/ba.dat",header=TRUE)
+
+average = rowMeans(ba)
+ba$average = average
+
+#Mean differences 
+difference <- ba$Wright - ba$Mini
+ba$difference = difference
+mean_difference = mean(difference)
+
+#Confidence interval
+lower <- mean_difference - 1.96*sd(ba$difference)
+upper <- mean_difference + 1.96*sd(ba$difference)
```

Comment:
Good approach here for copying your script form the R file to this part of the RMarkdown. Well done!